src/util/FactorUtil.js

```diff
@@ -90,12 +90,18 @@ define(['okta'], function (Okta) {
       iconClassName: 'mfa-onprem',
       sortOrder: 10
     },
+    'EMAIL': {
+      label: 'factor.email',
+      description: '',
```

Comment:
This description shows when you need to select a factor to enroll. Since email is auto-enrolled, it doesn't really apply here.